src/prefect/orion/schemas/core.py

```diff
@@ -389,7 +390,6 @@ class TaskRun(ORMBaseModel):
         description="A dynamic key used to differentiate between multiple runs of the same task within the same flow run.",
     )
     cache_key: Optional[str] = Field(
-        default=None,
```

Comment:
Why are we removing the default value? This seems incorrect.

Reply:
It is incorrect.